pyglet/image/__init__.py

```diff
@@ -1203,17 +1259,19 @@ class Texture(AbstractImage):
 
     """
 
-    region_class = None  # Set to TextureRegion after it's defined
```

Comment:
Lets leave these local variables as they are. I don't see much benefit from typing information here, as they're internal defaults. It just adds more clutter, IMO.